PAO-Project/src/Curs/Materie.java

```diff
@@ -0,0 +1,39 @@
+package Curs;
```

Comment:
[Fix] in java toate pachetele se scriu cu litera mica, doar numele de clasa incep cu litera mare